index.html

```diff
@@ -0,0 +1,1049 @@
+<!DOCTYPE  html> 
```

Comment:
- [ ] We understand that the webpages change very often. One suggestion is to try to archive the webpage or take some screenshots for us to compere. There are details that always stay the same as the display of the editor's pick logo and text, the way that "see all features" in the first article is slightly bigger than the other pictures. I am going to point out those particular details here. Kindly fix the title layout and the first picture layout (I am just referring to image size).

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83209815-d9991300-a11e-11ea-96c8-66104aea8194.png) | ![image](https://user-images.githubusercontent.com/57414789/83209776-be2e0800-a11e-11ea-9731-a1b44907ae0d.png)

Reply:
Thanks for your comments. They helped me a lot.